src/core/fibers.h

```diff
@@ -12,20 +12,9 @@
 
 namespace dfly {
 
-using util::fb2::Barrier;
-using util::fb2::BlockingCounter;
-using util::fb2::CondVar;
-using util::fb2::Done;
-using util::fb2::EventCount;
 using util::fb2::Fiber;
```

Comment:
If we are replacing the uses of `dfly::XXXX` with `util::fb2::XXXX` why don't you remove the rest of the header as well ?

Reply:
because I do not have time and I do it in chunks